packages/tx/typedoc.js

```diff
@@ -4,7 +4,10 @@ module.exports = {
   plugin: 'typedoc-plugin-markdown',
   readme: 'none',
   gitRevision: 'master',
-  exclude: 'test/**/*.ts',
+  exclude: [
+    'src/util.ts',
```

Comment:
Why do we exclude this?

Reply:
The API of this feels too unfinished to me and I would like to keep this internal for now and give this some more thought and have some more discussions around it. I think no one every really reviewed these code parts (including me) and this would expose a huge new API surface.

Some things I would like to discuss (but which doesn't need to be discussed now, just for the mention):

- Is this the best way to expose this functionality or might there be better - eventually more integrated - solutions (e.g. to re-bind the methods to the affected tx classes, just as some first idea)?
- Relationship of this `getAccessListData()` (returning `AccessListJSON` as one of two items) and the `getAccessListJSON()` method
- Should `getDataFeeEIP2930()` be exposed (I have the base feeling that methods which need a `Common` as an input often point to some suboptimality in design - we had these cases in the past - can't really point towards a concrete solution thought atm yet)? 

I will actually open this as an issue, otherwise it will likely be too tempting to start the discussion here. 😄